src/utils/walletIntegration/wallets/walletConnect.ts

```diff
@@ -177,6 +177,14 @@ class WalletConnectIntegration implements WalletIntegrationInterface {
     // WalletConnect balance retrieval logic
   }
 
+  async addAsset(assetId: string, symbol: string, logo: string): Promise<void> {
+    console.log('Currently only Goby has support for adding an asset programatically')
```

Comment:
toast? also, I'm pretty sure Chia has it, too

Reply:
I pasted the code from Hoogii. Will be built out when I work on WalletConnect again